Translation corrections (#479)
07th-mod/higurashi-console-arcs: Update/omot_022.txt

//rそして皮肉なことに、…これは梨花ちゃまを狙う敵が、作戦の発動のためには彼女の『明確な死』を必要とすることを裏付ける、明らかな証拠でもあったのだ――。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして皮肉なことに、…これは梨花ちゃまを狙う敵が、作戦の発動のためには彼女の『明確な死』を必要とすることを裏付ける、明らかな証拠でもあったのだ——。",

> 			NULL, "And ironically enough... that proved that the enemies targeting Rika-chama needed \"proof of death\" in order to carry out their plans.", Line_Normal);

Reviewer: ```suggestion
			NULL, "And ironically enough... that proved the enemies targeting Rika-chama needed \"proof of death\" in order to carry out their plans.", Line_Normal);
```

		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>鑑識</color>", NULL, "<color=#f5e6d3>Forensic Investigator</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 0, "ps2/00/223000035", 540, TRUE);
		OutputLine(NULL, "しかもその傷の周辺の化膿部に、明らかに北条鉄平とは違う毛髪が一本発見された」",

> 				NULL, " Moreover, a single hair was found in the wound's pus that clearly didn't belong to Teppei Houjou.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Moreover, a single hair was found in the suppurating wound that clearly didn't belong to Teppei Houjou.\"", Line_Normal);
```

	} else {
		ModPlayVoiceLS(4, 0, "ps3/s14/00/223000034", 256, TRUE);
		OutputLine(NULL, "「痕跡が組成された箇所、そして時間から逆算しても、残留反応がないのは不自然じゃ。",
				NULL, "\"Considering the location of the mark and calculating backward from that point in time, it's unnatural that there's no residual reaction.", GetGlobalFlag(GLinemodeSp));
		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>鑑識</color>", NULL, "<color=#f5e6d3>Forensic Investigator</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 0, "ps3/s14/00/223000035", 256, TRUE);
		OutputLine(NULL, "しかもその傷の周辺の化膿部に、明らかに北条鉄平とは違う毛髪が一本発見された」",

> 				NULL, " Moreover, a single hair was found in the wound's pus that clearly didn't belong to Teppei Houjou.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Moreover, a single hair was found in the suppurating wound that clearly didn't belong to Teppei Houjou.\"", Line_Normal);
```

			NULL, "\"No, please listen.", Line_WaitForInput);
	ModPlayVoiceLS(4, 4, "ps3/s14/04/220400075", 256, TRUE);
	OutputLine(NULL, "私ずっと眠っている間、あの時のことを何度も夢に見ました。",

> 			NULL, " I dreamed about it many times recently when I slept for days.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " I've had many dreams about back then.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fcdb77>沙都子</color>", NULL, "<color=#fcdb77>Satoko</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 4, "ps3/s14/04/220400129", 256, TRUE);
	OutputLine(NULL, "「あの声は、…もしかしたら梨花の声だったのかもしれないって、今なら思います。",
			NULL, "\"I think now... that might have been your voice, Rika.", Line_WaitForInput);
	ModPlayVoiceLS(4, 4, "ps3/s14/04/220400130", 256, TRUE);
	OutputLine(NULL, "…だから、あなたのことは、この身に代えても守ってみせます。",
			NULL, " ...And so, I'll give everything I have to protect you.", Line_WaitForInput);
	ModPlayVoiceLS(4, 4, "ps3/s14/04/220400131", 256, TRUE);
	OutputLine(NULL, "それが、私の贖罪と、生きがいなんですわ」",

> 			NULL, " That's my atonement and my reason to live.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " That's my atonement, and my reason to live.\"", Line_Normal);
```
